docs/content/configuration/creating_volumestorageclass.md

```diff
@@ -24,14 +24,16 @@ Use the `SpaceEfficiency` parameters for each storage system, as defined in [the
 
 - The IBM DS8000 family `pool` value is the pool ID and not the pool name as is used in other storage systems.
 - Be sure that the `pool` value is the name of an existing pool on the storage system.
```

Comment:
please sync with the [latest example yamls changes](https://github.com/IBM/ibm-block-csi-driver/pull/472)

Reply:
done